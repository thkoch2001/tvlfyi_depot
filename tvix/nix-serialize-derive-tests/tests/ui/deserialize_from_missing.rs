use nix_compat_derive::NixDeserialize;

#[derive(NixDeserialize)]
#[nix(from = "u64")]
pub struct Test;

fn main() {}
