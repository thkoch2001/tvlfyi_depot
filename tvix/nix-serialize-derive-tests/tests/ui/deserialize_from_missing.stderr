error[E0277]: the trait bound `Test: From<u64>` is not satisfied
 --> tests/ui/deserialize_from_missing.rs:4:14
  |
4 | #[nix(from = "u64")]
  |              ^^^^^ the trait `From<u64>` is not implemented for `Test`
